code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -894,6 +894,19 @@
 	. = TRUE
 	..()
 
+/datum/reagent/toxin/acid/fluantiacid
+	name = "Fluoroantimonic acid"
+	description = "Fluoroantimonic acid is the strongest corrosive chemical substance ever discovered."
+	color = "#5050FF"
+	toxpwr = 15
+	acidpwr = 200000000000000.0 //Yes, 200 quintillion, Fluoroantimonic acid is 2x10^19 times stronger than sulphuric acid. Don't worry, the antimony will be quite rare.
```

Comment:
A: WHY IN THE NAME OF GOD B: What does this do?

Reply:
> This pr will need to be tested, by you, I recommend getting a git client set up. As far as the acidpwr goes, we don't intend to model real life quite that closely, you'll want to set it to something that describes the effect you want, not necessarily the ratio between the strengths of other mixes. Does that make sense?
> 
> I'd also like more details on why this will be good for the game, as I'm against adding more acids for little reason.

This is a draft to pitch an idea. It's missing 4/5ths of the code I added because I can't figure out mobile github. In the actual pr I'll be sure to explain a lot more about it